Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -1483,6 +1495,18 @@ private void SetTopParentIdsOrAncestors(InternalItemsQuery query, IReadOnlyColle
             query.Parent = null;
         }
 
+        [Obsolete("LibraryManager.GetChapters is deprecated, please use ChapterManager.GetChapters instead.")]
```

Comment:
The `[Obsolete]` attributes should be added at the interface level, nothing directly calls theses functions